src/Spanner/Session/CacheSessionPool.php

```diff
@@ -777,4 +764,63 @@ private function validateConfig()
             );
         }
     }
+
+    /**
+     * Delete the provided sessions.
+     *
+     * @param array $sessions
+     */
+    private function deleteSessions(array $sessions)
+    {
+        $calls = [];
+
+        foreach ($sessions as $session) {
+            $calls[] = $this->database->connection()
+                ->deleteSessionAsync([
+                    'name' => $session['name'],
+                    'database' => $this->database->name()
+                ]);
+        }
+
+        foreach ($calls as $call) {
+            $call->wait();
```

Comment:
I believe so - the tests were run using [gRPC 1.4.6](https://pecl.php.net/package-changelog.php?package=gRPC&release=1.4.6) which includes [persistent connections by default](https://github.com/grpc/grpc/pull/11878).